docs/introduction/Installation.md

```diff
@@ -8,6 +8,85 @@
 
 ## vcpkg
 
+## Compiling from Source - Ubuntu 16.04 LTS (Xenial Xerus) 
+Please run the following commands and read the comments as you go.
+You will notice that we update and upgrade quite frequently, this is to ensure that all dependancies are met at all times. You will also notice that we are installing some libraries from source, there are two reasons for this; firstly in the event that the library is not available through apt-get or secondly, because we want the very latest version (not the version available in apt-get).
+```
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get install cmake
+sudo apt-get install zlibc
+sudo apt-get install zlib1g
+sudo apt-get install zlib1g-dev
+sudo apt-get install libssl-dev
+sudo add-apt-repository ppa:ubuntu-toolchain-r/test
+sudo apt update
+sudo apt-get upgrade
+sudo apt-get install gcc-6 g++-6
```

Comment:
Can the older build-essential version of gcc (v4.4 I think) bootstrap the latest gcc? If so, it might be preferable to installing another compiler here.